portworx-k8-sched-snap/assets/create-vols.sh

```diff
@@ -0,0 +1,6 @@
+ssh -o strictHostKeyChecking=no node01 "pxctl volume create testvol"
+
+ssh -o strictHostKeyChecking=no node02 "pxctl volume create local_volume --nodes=LocalNode"
+
+ssh -o strictHostKeyChecking=no node03 "pxctl volume create ha_volume --nodes=LocalNode --repl3"
```

Comment:
need space or `=` in --repl3

Reply:
This is fixed already -->
vijin@kodekloud:~/portworx/katacoda-scenarios/portworx-k8-sched-snap$ cat create-vols.sh | grep repl
ssh -o strictHostKeyChecking=no node03 "pxctl volume create ha_volume --nodes=LocalNode --repl=3"
vijin@kodekloud:~/portworx/katacoda-scenarios/portworx-k8-sched-snap$